crates/core/tedge_actors/src/runtime.rs

```diff
@@ -493,30 +496,36 @@ mod tests {
         );
     }
 
-    // TODO: An actor panic doesn't print the way I want
-    // #[tokio::test]
-    // async fn actor_panics() {
-    //     let (mut actions_sender, mut events_receiver, runtime_request_sender, ra) = init();
-    //     let (mut input_sender, mut output_receiver, actor) =
-    //         create_actor(Panic {}, &runtime_request_sender);
-
-    //     actions_sender
-    //         .send(RuntimeAction::Spawn(Box::new(actor)))
-    //         .await
-    //         .unwrap();
-
-    //     let wait_for_messages = async {
-    //         output_receiver.next().await;
-
-    //         while let Some(event) = events_receiver.next().await {
-    //             dbg!(&event);
-    //         }
-    //         true
-    //     };
-
-    //     tokio::select! {
-    //         spawned_actor_event_received = wait_for_messages => assert!(spawned_actor_event_received, "The actor was not spawned"),
-    //         _ = ra.run() => panic!("The runtime actor finished unexpectedly")
-    //     };
-    // }
+    #[tokio::test]
+    async fn actor_panics() {
+        let (mut actions_sender, mut events_receiver, ra) = init();
+        let (_, _, panic_actor) = create_actor(Panic);
+        let (mut sender, _, echo_actor) = create_actor(Echo);
+
+        actions_sender
+            .send(RuntimeAction::Spawn(Box::new(panic_actor)))
+            .await
+            .unwrap();
+
+        actions_sender
+            .send(RuntimeAction::Spawn(Box::new(echo_actor)))
+            .await
+            .unwrap();
+
+        let wait_for_actors_to_panic = async {
+            while let Some(event) = events_receiver.next().await {
+                if matches!(event, RuntimeEvent::Aborted { .. }) {
+                    break;
+                }
+            }
+        };
+
+        tokio::select! {
+            _ = ra.run() => panic!("Runtime finished unexpectedly"),
+            _ = wait_for_actors_to_panic => {
+                sender.send(EchoMessage::String("hello".into())).await.expect("Expected the echo actor to be running and to receive a message");
+            }
+            _ = tokio::time::sleep(Duration::from_secs(1)) => panic!("Actor failed to panic` in time")
+        }
```

Comment:
```suggestion
        tokio::spawn(async { ra.run() });
        let foo = tokio::time::timeout(Duration::from_secs(1), wait_for_actors_to_panic).await;
        echo_sender
            .send(EchoMessage::String("hello".into()))
            .await
            .expect("Expected the echo actor to be running and to receive a message");
        assert_eq!(
            echo_receiver.next().await.unwrap(),
            EchoMessage::String("hello".into())
        );
```
Wondering if the readability of this test could be improved by doing something along these lines, instead of the relatively complex `select!`.